manifests/charts/dash/templates/job-init.yaml

```diff
@@ -4,7 +4,7 @@ kind: Job
 metadata:
   name: {{ include "dash.fullname" . }}-init
   annotations:
-    "helm.sh/hook": post-install
+    "helm.sh/hook": post-install,post-upgrade
```

Comment:
I had this locally and it's ringing a bell that we may have needed it. Any ideas why?
@jacobbeasley @kyle-intelletive

Reply:
With just post install, this job will only run after the initial helm install.

With both, it will run after the initial helm install and every helm upgrade.